Week08/src/Main.java

```diff
@@ -0,0 +1,122 @@
+import java.awt.*;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("Welcome to Imaginationland University. Choose one of the following options:");
+        System.out.println("1. Add Student");
+        System.out.println("2. Delete Student");
+        System.out.println("3. Retrieve all students of a certain age");
+        System.out.println("4. List students");
+        System.out.println("5. Exit");
+
+        List<Student> StudentList = new ArrayList<>();
+        //  SimpleDateFormat date_format = new SimpleDateFormat("YYYY-MM-DD");
+
+        StudentList.add(new Student("Jew", "Gasington", "1980-06-22", "M", 1984653));
+        StudentList.add(new Student("Floricica", "Mirosica", "2001-08-05", "F", 2365488));
+        StudentList.add(new Student("Jimmy", "Volmer", "1999-09-11", "M", 18523697));
+
+        String option = "";
+        option = sc.next();
+        sc.nextLine();
+
+
+        while (!option.equals("5")) {
+
+            switch (option) {
+
+                case ("1"):
+                    for (Student a : StudentList) {
+                        a.listStudent();
+                    }
+                    System.out.println("What is the student fist name?");
+                    String first_name = sc.nextLine();
```

Comment:
Please don't forget about naming conventions. Same goes for the other variables.